### ISSUE TITLE
How to add One more label to existing pretrained labels.

### ISSUE THREAD

**Issue by purnasai** [open-mmlab/mmsegmentation#950]
Hi,
how to add one more label to the existing pre-trained model labels.

**Ex:** I have a pspnet model trained on the Cityscapes dataset with 19 labels/classes in it. If I want to add one more label(ex:curb) to the model, such that it can predict 19+1 classes in the output image. The curb should also be detected along with the other 19 classes in the image. 

If there is any notebook in the repo, can you point me to it?
Thanks

**Comment by MengzhangLI** [comment]
Hi, I think the best solution is generating new labels of cityscapes datasets and training the model again. Existing model could not transfer to new dataset label.
